### PR TITLE
Implement paginated fetching of features

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

File: src/common/ui-components/Map/Map.tsx [hunk 1]
```diff
@@ -8,8 +8,7 @@ import MapGL, {
   FlyToInterpolator,
   TransitionInterpolator,
   EasingFunction,
-  TRANSITION_EVENTS,
-  ViewportProps
+  TRANSITION_EVENTS
```
Comment (marharyta) [inline src/common/ui-components/Map/Map.tsx]: this will not work with the current Map setup

File: src/common/ui-components/Map/Map.tsx [hunk 2]
```diff
@@ -94,11 +93,16 @@ const Map: React.FC<MapProps> = ({
     minZoom: minZoomLevel,
     maxZoom: maxZoomLevel
   });
+
   const mapRef = useRef<StaticMap>();
   const renderPin = (
     pointFeature: PointFeature<GeoJsonProperties>,
     id: number | string
   ) => {
+    if (pointFeature.geometry.type !== 'Point') {
```
Comment (marharyta) [inline src/common/ui-components/Map/Map.tsx]: Good catch

**Comment by marharyta** [comment]
Mar-23-2020 11-46-54

**Comment by ahie** [comment]
Not sure where you're getting that behavior. At least it works as intended in the test environment (https://ahti-ui-qa-r-feature-ahti-f3zihz.test.kuva.hel.ninja)

File: src/common/ui-components/Map/Map.tsx (hunk 1 above)
Comment (ahie) [reply src/common/ui-components/Map/Map.tsx]: It's an unused import.

**Comment by marharyta** [comment]
@ahie when I pull locally

**Comment by marharyta** [comment]
@ahie I have checked when i 'yarn start' it works as intended

**Comment by marharyta** [comment]
@ahie can you reproduce it in storybook?

File: src/common/ui-components/Map/Map.tsx (hunk 1 above)
Comment (marharyta) [reply src/common/ui-components/Map/Map.tsx]: oh, true, I didn`t see the deletion in the bottom

**Pull request merged**